common/src/main/resources/index-mappings/ml_index_insight_container.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
It seems this schema is not consistent with code, `content`, `task_type` etc are not defined.

Reply:
No. This is for class `IndexInsightContainer` while `content, task_type` should be in `indexInsight`